apps/web/src/services/cost-explorer/pages/admin/AdminAdvancedSettingsCurrencyConverterPage.vue

```diff
@@ -1,47 +1,183 @@
 } from '@cloudforet/mirinae';
 
-import { store } from '@/store';
 
-import { useDomainStore } from '@/store/domain/domain-store';
-// import { usePreferencesStore } from '@/store/preferences/preferences-store';
+import { CURRENCY } from '@/store/modules/display/config';
+import type { Currency } from '@/store/modules/display/type';
+import { usePreferencesStore } from '@/store/preferences/preferences-store';
 
-import type { PageAccessMap } from '@/lib/access-control/config';
+import ScopedNotification from '@/common/components/scoped-notification/ScopedNotification.vue';
+import ErrorHandler from '@/common/composables/error/errorHandler';
 
-// const domainConfigStore = usePreferencesStore();
-// const domainConfigGetters = domainConfigStore.getters;
-const domainStore = useDomainStore();
+import type { UnifiedCostConfig } from '@/services/advanced/types/preferences-type';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const storeState = reactive({
-    domainId: computed<string>(() => domainStore.state.domainId),
-    domainName: computed<string>(() => domainStore.state.name),
-    domainConfig: computed(() => domainStore.state.config),
-    pageAccessPermissionMap: computed<PageAccessMap>(() => store.getters['user/pageAccessPermissionMap']),
+const DEFAULT_AGGREGATION_DATE = 15;
+
+const domainConfigStore = usePreferencesStore();
+const domainConfigGetters = domainConfigStore.getters;
+
+const exchangeRateSourceOptions = ['Yahoo Finance!'];
+
+const state = reactive({
+    loading: false,
+    originUnifiedCostConfig: computed<UnifiedCostConfig|undefined>(() => domainConfigGetters?.unifiedCostConfig),
+    isSaveChangesButtonDisabled: computed<boolean>(() => {
+        if (formState.aggregationDate === '') {
+            return true;
+        }
+        const aggregationDate:number = (typeof formState.aggregationDate === 'string') ? parseInt(formState.aggregationDate) : formState.aggregationDate;
+
+        let isSameAggregationDate = aggregationDate === state.originUnifiedCostConfig?.aggregation_day;
+        if (aggregationDate < 1 || aggregationDate > 31) {
+            isSameAggregationDate = true;
+        }
+        return (formState.currency === state.originUnifiedCostConfig?.currency
+        && isSameAggregationDate
+        && formState.exchangeRateSource === state.originUnifiedCostConfig?.exchange_source
+        && formState.lastDayOfMonth === state.originUnifiedCostConfig?.is_last_day);
+    }),
+});
+
+const formState = reactive<{
+    currency: Currency;
+    aggregationDate: number|string;
+    exchangeRateSource: string;
+    lastDayOfMonth: boolean;
+}>({
+    currency: state.originUnifiedCostConfig?.currency ?? CURRENCY.KRW,
+    aggregationDate: state.originUnifiedCostConfig?.aggregation_day ?? DEFAULT_AGGREGATION_DATE,
+    exchangeRateSource: state.originUnifiedCostConfig?.exchange_source ?? 'Yahoo Finance!',
+    lastDayOfMonth: state.originUnifiedCostConfig?.is_last_day ?? false,
+});
```

Comment:
quibble: Is 'Yahoo Finance' set as default?

Reply:
Yes, It has not been confirmed how it will be added in the future, so it has been implemented as currently required.